python/lsst/ts/imsim/opdMetrology.py

```diff
@@ -140,13 +141,15 @@ def set_wgt_and_field_xy_of_gq(self, inst_name: str) -> None:
             self.set_default_lsst_wfs_gq()
             return
 
-        instrument_path = getConfigDirOfc() / inst_name
+        weight_dir_path = getConfigDirOfc() / "image_quality_weights"
```

Comment:
I am just checking now, I am not sure where these weights are used in ts_imsim. I was checking and I can't find anywhere where we use self.wt, but I may be wrong. Do you know?

Reply:
It is used when calculating the fwhm and pssn with the OPD. See `calc_gq_value` in this module.